src/dispatch/incident/flows.py

```diff
@@ -62,7 +54,10 @@
 
 
 def get_incident_participants(incident: Incident, db_session: SessionLocal):
```

Comment:
I think it'd be helpful to have a return type for this one since `plugin.instance.get` wont resolve the correct types.

```suggestion
def get_incident_participants(
    incident: Incident, db_session: SessionLocal
) -> tuple[list[Participant | None], list[Participant | None]]:
```

Reply:
Committed changes.